pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -51,7 +51,9 @@ stdenv.mkDerivation rec {
     sha256 = "15iw7982g6vc4jy1l9kk1z9sl5bm1bdbwr74y7nvwjs1nffhig7f";
   };
 
-  nativeBuildInputs = [ makeWrapper python python.pkgs.sphinx python.pkgs.sphinx_rtd_theme pkg-config flex bison meson ninja ]
+  depsBuildBuild = [ buildPackages.stdenv.cc binutils perl python3 python3Packages.sphinx python3Packages.sphinx_rtd_theme ];
```

Comment:
things are a little bit trickier for `stdenv.cc`, while it is not spliced, we have no top-level attr that is equivalent to it.

Reply:
it's fine imo. The point is more that pkgsBuildBuild or depsBuildBuild is a bit clearer than stacking buildPackages several times.

Reply:
I don't think the Python stuff should be moved here — it works fine for me in `nativeBuildInputs`, and there shouldn't be anything different about `depsBuildBuild` for Python stuff since it doesn't have a target platform.  Similarly, `perl` should be a native build input.

Reply:
And what do we need binutils for?  That should be covered by stdenv.cc.

Reply:
binutils is for `strings`

Reply:
binutils is for `strings`

Reply:
`strings` seems to be available in the build even without binutils for me?